### DISCUSSION TITLE
CLI UI

### DISCUSSION THREAD

**Discussion by avarama** [purpleteam-labs/purpleteam#111] [Q&A] [answered]
Hello again,

The CLI UI is a really nice ncurses-like dashboard of boxes, but keystrokes to navigate it sometimes give me problems. I remember reading something about the option to use really basic CLI UI, like plain logs instead of the nice boxed dashboard, but now I cannot find the info. It was something like `--no-ui`. I'm talking about the CLI component. The one actually running the tests. Thanks!

**Comment by binarymist** [comment, marked as answer]
Hi @avarama.

Yeah the idea is that the `cUi` does the job of manually starting and monitoring _Test Runs_ and `noUi` is for running the CLI without a UI which is good if you want to run the CLI as part of a build. You are probably looking for the [`uI`](https://github.com/purpleteam-labs/purpleteam#ui) section on the CLI README?

We're currently in the middle of a [large upgrade](https://github.com/purpleteam-labs/purpleteam/issues/107) which will probably see these config values change slightly, just a heads up that next release may mean you have to tweak the value.
